photon-ml/src/main/scala/com/linkedin/photon/ml/cli/game/scoring/Params.scala

```diff
@@ -82,10 +86,10 @@ object Params {
     val defaultParams = Params()
     val parser = new OptionParser[Params]("GLMix-Scoring-Params") {
       opt[String]("input-data-dirs")
-        .required()
-        .text("Input directories of data to be scored. Multiple input directories are also accepted if they are " +
-          "separated by commas, e.g., inputDir1,inputDir2,inputDir3.")
-        .action((x, c) => c.copy(inputDirs = x.split(",")))
+          .required()
+          .text("Input directories of data to be scored. Multiple input directories are also accepted if they are " +
+              "separated by commas, e.g., inputDir1,inputDir2,inputDir3.")
+          .action((x, c) => c.copy(inputDirs = x.split(",")))
```

Comment:
Thanks for pointing it out. 

The continuation indent could be confusing sometimes, for example, for method/class's parameters the continuation indent is recommended to be 4-space (https://github.com/databricks/scala-style-guide#indent), while it looks like for most other cases it should be 2-space, as you suggested. But unfortunately Intellij doesn't allow us to set the continuation indent case by case, so we need to pay extra attention to it each time.